packages/mime/README.md

```diff
@@ -0,0 +1,86 @@
+# `@wordpress/mime`
+
+A set of mime type-related helper utilities, wrapping [`mime/lite`](https://www.npmjs.com/package/mime).
```

Comment:
Do we really need a wrapper package, why not use `mime` directly to avoid a new public API / package and maintenance burden. 

Mostly asking because the content of the package seems very small.

Reply:
Sure I can reuse it directly 👍